includes/utils.php

```diff
@@ -1121,3 +1123,30 @@ function post_args_allow_list( $post_args ) {
 
 	return array_intersect_key( $post_args, array_flip( $allowed_post_keys ) );
 }
+
+/**
+ * Make a remote GET request.
+ *
+ * Wrapper function for wp_remote_get() and vip_safe_wp_remote_get(). The order
+ * of parameters differs from vip_safe_wp_remote_get() to promote the arguments array
+ * to the second parameter.
+ *
+ * See {@see http://developer.wordpress.org/reference/classes/WP_Http/request/ WP_Http::request} for $args defaults.
+ *
+ * @param  string $url       The URL to request.
+ * @param  array  $args      Optional. An array of arguments to pass to wp_remote_get()/vip_safe_wp_remote_get().
+ * @param  mixed  $fallback  Optional. Fallback value to return if the request fails. Default false. VIP only.
```

Comment:
So I know we are setting this to `false` currently but in reading the [documentation](https://docs.wpvip.com/technical-references/code-quality-and-best-practices/retrieving-remote-data/#h-vip_safe_wp_remote_get), sounds like this is expected to be a string (default would be an empty string).

Reply:
Fixed in 7b156bb1f4239aab8f6a9e5a171f6d7124ea0cce